velox/vector/tests/VectorTest.cpp

```diff
@@ -576,6 +577,99 @@ class VectorTest : public testing::Test, public test::VectorTestBase {
     testCopy(lazy, level - 1);
   }
 
+  static void testSlice(
+      const VectorPtr& vec,
+      int level,
```

Comment:
nit: what does level denote here ?

Reply:
It's the number of wrap we will put on the test data.  Same thing is done in `testCopy`